contracts/modules/DailyLimitModule.sol

```diff
@@ -1,19 +1,18 @@
-pragma solidity 0.4.19;
-import "../Extension.sol";
-import "../GnosisSafe.sol";
+pragma solidity 0.4.23;
+import "../Module.sol";
+import "../ModuleManager.sol";
+import "../OwnerManager.sol";
+import "../Enum.sol";
 
 
-/// @title Daily Limit Extension - Allows to transfer limited amounts of ERC20 tokens and Ether without confirmations.
+/// @title Daily Limit Module - Allows to transfer limited amounts of ERC20 tokens and Ether without confirmations.
 /// @author Stefan George - <stefan@gnosis.pm>
-contract DailyLimitExtension is Extension {
+contract DailyLimitModule is Module {
 
-    string public constant NAME = "Daily Limit Extension";
+    string public constant NAME = "Daily Limit Module";
     string public constant VERSION = "0.0.1";
     bytes4 public constant TRANSFER_FUNCTION_IDENTIFIER = hex"a9059cbb";
```

Comment:
Consider calc. identifier inline for transparency: `bytes4 public constant CALL_PREFIX = bytes4(keccak256("callGasRelay(address,uint256,bytes32,uint256,uint256,address)"));`